### ISSUE TITLE
Doesn't work with cross-platform Azure CLI

### ISSUE THREAD

**Issue by liamdawson** [arafato/azurite#126]
Tried:

```
az storage container create --name testcontainer --connection-string 'BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;'
```

Outcome:

```
az storage container create: error: 'AccountName'
```

Version: 1.8.0 (Docker)

Azure CLI version:

```
± az --version
azure-cli (2.0.19)

acr (2.0.13)
acs (2.0.17)
appservice (0.1.18)
backup (1.0.1)
batch (3.1.5)
batchai (0.1.1)
billing (0.1.5)
cdn (0.0.9)
cloud (2.0.8)
cognitiveservices (0.1.8)
command-modules-nspkg (2.0.1)
component (2.0.7)
configure (2.0.11)
consumption (0.1.5)
container (0.1.11)
core (2.0.18)
cosmosdb (0.1.13)
dla (0.0.12)
dls (0.0.15)
eventgrid (0.1.4)
extension (0.0.4)
feedback (2.0.6)
find (0.2.7)
interactive (0.3.10)
iot (0.1.12)
keyvault (2.0.12)
lab (0.0.11)
monitor (0.0.10)
network (2.0.16)
nspkg (3.0.1)
profile (2.0.14)
rdbms (0.0.7)
redis (0.2.9)
resource (2.0.16)
role (2.0.13)
servicefabric (0.0.4)
sql (2.0.13)
storage (2.0.17)
vm (2.0.16)
```

**Comment by liamdawson** [comment]
The following connection string gave me a 'Not Implemented' instead, which makes me think I'm onto the right track:

```
az storage container create --name 'test' --public-access off --connection-string 'DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/'
```

**Comment by liamdawson** [comment]
Now I'm receiving a `Cannot read property 'query' of undefined` error with a new connection string:

```
az storage container create --name 'test' --public-access off --connection-string 'DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/' --debug
```

**Comment by liamdawson** [comment]
After checking the routing, I found that I needed to add `devstoreaccount1` back to the blob endpoint. Final, working connection string:

```
DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;
```

**Comment by arafato** [comment]
thanks a lot @liamdawson really great work! Just merged your PR. I deleted the tmp folder from the repository since Azurite automatically creates the folder if it does not exist. So there is no need to keep it in the repo IMO. This makes it more convenient to run the clean script (npm run clean) which simply deletes the entire folder (which I renamed to azurite-workspace). This folder is also automatically created if you run Azurite without any location parameter (npm start).
